src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/CodeActions/CodeActionEndpoint.cs

```diff
@@ -176,8 +176,12 @@ internal async Task<RazorCodeActionContext> GenerateRazorCodeActionContextAsync(
         private async Task<IEnumerable<RazorCodeAction>> GetCSharpCodeActionsAsync(RazorCodeActionContext context, CancellationToken cancellationToken)
         {
             var csharpCodeActions = await GetCSharpCodeActionsFromLanguageServerAsync(context, cancellationToken);
-            var filteredCSharpCodeActions = await FilterCSharpCodeActionsAsync(context, csharpCodeActions, cancellationToken);
+            if (csharpCodeActions is null || !csharpCodeActions.Any())
```

Comment:
Nit: I feel like `csharpCodeActions.Length == 0` is a bit clearer here unless we have to worry about `null` list entries.

Reply:
https://dev.azure.com/dnceng/public/_build/results?buildId=823668&view=logs&j=1f132584-ab08-5fbf-e5d0-5685de454342&t=76c675cd-a096-5dd5-8431-7aa49ff610be&l=147

Reverting...